research-app/src/App.vue

```diff
@@ -26,14 +26,14 @@
           v-bind:imageset="imageset"
         />
       </div>
-      <div id="catalogs-container" v-if="haveCatalogs">
+      <div id="spreadsheets-container" v-if="haveTableLayers">
         <div class="display-section-header">
-          <label>Catalogs</label>
+          <label>Catalogs and Spreadsheets</label>
         </div>
-        <catalog-item
-          v-for="catalog of hipsCatalogs"
-          v-bind:key="catalog.name"
-          v-bind:catalog="catalog"
+        <spreadsheet-item
+          v-for="layer of layers"
```

Comment:
So, I the state item here should be named something like `tableLayers` instead of just `layers`. Looking farther down in the code, it looks like that's actually the name used in the Vuex store, and I definitely think we should tend to preserve those names to make things overall more consistent.